CHANGELOG.md

```diff
@@ -32,6 +32,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Austin Emmons](https://github.com/atreat)
   [#6742](https://github.com/CocoaPods/CocoaPods/issues/6742) 
 
+* Add ipc command `podfile_json` converts a Podfile to Json
```

Comment:
please add a period and 2 spaces to the changelog entry, in addition to capitalizing `JSON`